### DISCUSSION TITLE
BotUpdater.ini changes don't immediately take effect when stopping and restarting botupdater.py

### DISCUSSION THREAD

**Discussion by ifmvaiea** [cyberjunky/3commas-cyber-bots#199] [General]
Hello,

When I make a simple change in botupdater.ini, like removing volatility-24h values and adding percent-change-7d values, my deal pairs don't update for my bot when saving the ini file, stopping and restarting botupdater.py. Same thing if I remove all values and just go with number of pairs using cmc-rank. There is no update for my bot back to the unfiltered pairs.

Is there a cache that stops the update from happening or does one of the other scripts override the update time for BotUpdater? The data from MarketCollector should be the same but I'm just using different settings for the deal pairs.

Thank you!

**Comment by amargedon** [comment]
No, there is no cache involved. Script will compare the current pairs against the fetched and filtered pairs from the market collector database.

Are you sure you did edit the configuration file that was used by the script? You could activate the debug option to get more detailed logging, maybe that would give some more insight in what is happening.
